src/EncompassRest/LoanPipeline/LoanPipelineCursor.cs

```diff
@@ -23,59 +23,53 @@ internal LoanPipelineCursor(EncompassRestClient client, string cursorId, int cou
             Fields = fields ?? Array<string>.Empty;
         }
 
-        public Task<LoanPipelineData> GetItemAsync(int index) => GetItemAsync(index, CancellationToken.None);
+        public Task<LoanPipelineData> GetItemAsync(int index) => GetItemAsync(index, null, CancellationToken.None);
 
-        public async Task<LoanPipelineData> GetItemAsync(int index, CancellationToken cancellationToken)
+        public Task<LoanPipelineData> GetItemAsync(int index, IEnumerable<string> itemFields) => GetItemAsync(index, itemFields, CancellationToken.None);
+
+        public async Task<LoanPipelineData> GetItemAsync(int index,IEnumerable<string> itemFields, CancellationToken cancellationToken)
         {
             Preconditions.GreaterThanOrEquals(index, nameof(index), 0);
             Preconditions.LessThan(index, nameof(index), Count, nameof(Count));
 
-            var data = await GetItemsInternalAsync(index, 1, cancellationToken).ConfigureAwait(false);
+            var data = await GetItemsInternalAsync(index, 1,itemFields, cancellationToken).ConfigureAwait(false);
             return data[0];
         }
 
-        public Task<List<LoanPipelineData>> GetItemsAsync(int start, int count) => GetItemsAsync(start, count, CancellationToken.None);
+        public Task<List<LoanPipelineData>> GetItemsAsync(int start, int count) => GetItemsAsync(start, count,null, CancellationToken.None);
```

Comment:
add space between arguments

I think `count` should be renamed to `limit` and be a nullable int. Thoughts?